backend/src/main/java/com/ethan5/service/ArtistService.java

```diff
@@ -25,7 +25,7 @@ public class ArtistService {
 
     public List<Artist> readTopArtists(String id, String bearerToken) {
         String url = "https://api.spotify.com/v1/me/top/artists?limit=3";
-        String token = bearerToken.substring(7);
+        String token = bearerToken.substring("Bearer ".length());
```

Comment:
Shouldn't have to include this change, magic number should be suppressed.

Reply:
Honestly I like it better this way, it's more obvious what the code is doing. But yeah initially I was using a checkstyle with magic number lol